changelog/dmd.dip25-default.dd

```diff
@@ -0,0 +1,15 @@
+`-preview=dip25` has been enabled by default
+
+Deprecation warnings for [DIP25](http://dlang.org/dips/25) violations have been enabled since 2.092.
+Starting with this release, it will report errors, unless the `-revert=dip25` switch is used.
+Using the switch (or its short version `-dip25`) is now deprecated.
+
+---
+ref int escapeRef(ref int x) {return x;}
```

Comment:
I tried this out, just this code doesn't emit any deprecation nor error with -preview=dip25 on dmd v2.102.0